### PR TITLE
feat: update to the latest forma and new tokens

### PR DESCRIPTION
**Pull request by suevalov**
* Bumps Forma36 to the latest version
* Update all tokens values (using codemod)

### CODE REVIEW

**Review by kdamball — COMMENTED** [review]
The previous naming vs current values are a bit confusing e.g: `ElementMid` < `TextMid` but overall code looks fine. I guess `colorPrimary` doesn't need changing?

I would manual check also on how apps look with these new values @fabe

**Comment by colomolo** [comment]
Checked thess apps (don't have access to all of them):
- [ ] + AI image tagging
- [ ] Brandfolder
- [ ] + Bynder
- [ ] + Cloudinary
- [ ] Commerce layer
- [ ] commercetools
- [ ] + Dropbox
- [ ] Frontify
- [ ] Gatsby Cloud
- [ ] Google Analytics
- [ ] + Image Focal Point
- [ ] + Jira
- [ ] + Knowledge base
- [ ] Mux
- [ ] Netlify
- [ ] + Optimizely
- [ ] Saleor Commerce
- [ ] Shopify
- [ ] + Smartling
- [ ] + Typeform

File: apps/optimizely/src/EditorPage.spec.js [hunk 1]
```diff
@@ -13,6 +13,9 @@ describe("EditorPage", () => {
   beforeEach(() => {
     sdk = mockProps.sdk;
     sdk.entry.fields.experimentId.onValueChanged = fn => () => {};
+
+    Date.now = jest.fn(() => 100);
```
Comment (kdamball) [inline apps/optimizely/src/EditorPage.spec.js:17]: Is there a reason for this change? 🤔

File: packages/dam-app-base/src/AppConfig/AppConfig.tsx [hunk 2]
```diff
@@ -185,7 +185,7 @@ export default class AppConfig extends React.Component<Props, State> {
     );
   }
 
-  onParameterChange = (key: string, e: React.ChangeEvent<HTMLInputElement>) => {
+  onParameterChange = (key: string, e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement> | React.ChangeEvent<HTMLSelectElement>) => {
```
Comment (kdamball) [inline packages/dam-app-base/src/AppConfig/AppConfig.tsx:188]: Same thing here - do we support more elements? I dont see an equivalent change in HTML? I still see the Select tag

**Review by kdamball — COMMENTED** [review]
Left a bunch of minor questions but LGTM so far as long as tests pass

**Review by andipaetzold — CHANGES_REQUESTED** [review]
Please recreate the `package-lock.json` files with npm 6. The files were (accidentally?) migrated to `lockfileVersion` 2.

I am also happy to upgrade the whole repo to npm 7 but I don't think this PR is the right place for that.

File: apps/optimizely/src/EditorPage.spec.js (hunk 1 above)
Comment (colomolo) [reply apps/optimizely/src/EditorPage.spec.js:17]: Tests failed due to tricky date and random number mock logic inside the Dropdown component:
https://github.com/contentful/forma-36/blob/99609176a0604fd0f6b62753cf45a246cdbb48b6/packages/forma-36-react-components/src/components/Dropdown/Dropdown.test.tsx#L11-L14

Ids are generated based on random number, so needed to stub it here too. Checked this in with @suevalov

Comment (kdamball) [reply apps/optimizely/src/EditorPage.spec.js:17]: Good to know, thanks!

File: packages/dam-app-base/src/AppConfig/AppConfig.tsx (hunk 2 above)
Comment (colomolo) [reply packages/dam-app-base/src/AppConfig/AppConfig.tsx:188]: Type error comes from `TextField` using `onParameterChange` handler:
https://github.com/contentful/apps/pull/336/files/67fad8b43f327d24a5cb9bcfcf8fe2c868d3f577#diff-6a90407ca4cc688bff8f85eecaa40447660585c61bc175d6a1ec7b414c472afaR261

Comment (kdamball) [reply packages/dam-app-base/src/AppConfig/AppConfig.tsx:188]: Good call. Maybe then removing the `HTMLInputElement`?

Comment (colomolo) [reply packages/dam-app-base/src/AppConfig/AppConfig.tsx:188]: But this types comes from forma type definition:
https://github.com/contentful/forma-36/blob/master/packages/forma-36-react-components/src/components/TextField/TextField.tsx#L35

**Review by kdamball — APPROVED** [review]
LGTM - but haven't done any UI check

**Pull request merged**